src/Service/ExecuteScheduleCommand.php

```diff
@@ -25,16 +25,21 @@ class ExecuteScheduleCommand
     /** @var string */
     private $logsDir;
 
+    /** @var string */
+    protected $projectDir;
```

Comment:
Why this is protected ? The logdir above is in private.

Reply:
My bad, you are right :)